avocado/utils/cpu.py

```diff
@@ -185,7 +185,9 @@ def get_family():
     arch = get_arch()
     if arch == 'x86_64' or arch == 'i386':
         if get_vendor() == 'amd':
-            raise NotImplementedError
+            # FIXME
+            logging.warning("Not implemented: Unknown cpu family for amd")
+            return family
```

Comment:
I'm not sure if this the right path to follow. Or I'm missing something.

As a library, users expect an exception instead of a None returning without the reason.

IMO, we should keep the exception, if this is not supported and users should use try...except to proper handle this.

Reply:
yep, I agree, thought of a quick fix to workaround the issue.
will send a PR on avocado-vt side to handle it better..